DVCSMP-3242 Update momentary-button-tile.groovy

vseven opened the pull request:
Sorry @workingmonk  ....too many copy and pastes in the last 24 hours...

Commit:
Update momentary-button-tile.groovy

Sorry @workingmonk ....too many copy and pastes in the last 24 hours...

workingmonk commented:
no worries. your contribution is greatly appreciated 🎉

workingmonk commented on devicetypes/smartthings/momentary-button-tile.src/momentary-button-tile.groovy:
```diff
@@ -29,10 +29,12 @@ metadata {
+	tiles(scale: 2){
+		multiAttributeTile(name:"switch", type: "generic", width: 6, height: 4){
+			tileAttribute("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState("off", label: 'Push', action: "momentary.push", backgroundColor: "#ffffff", nextState: "on")on")
```
extra `on")` at the end

vseven replied:
GAH!

Commit:
Update momentary-button-tile.groovy

So when I edited the file it ended with the first "on") and I didn't even see the extra.  Had to go to the next line and backspace a bunch until it erased.  GitHub editor failed me.  

@workingmonk - fixed
